### PR TITLE
Jvenstad/minors

### PR DESCRIPTION
**Pull request by jonmv**
@jobergum please take a look :]

### CODE REVIEW

File: boolean-search/src/main/application/searchdefinitions/ad.sd [hunk 1]
```diff
@@ -13,8 +14,10 @@ search ad {
         arity: 8
```
Comment (jonmv) [inline boolean-search/src/main/application/searchdefinitions/ad.sd]: Nested : are not allowed?

**Comment by kkraune** [comment]
@jonmv please consider whether to close this / create new / whatever - seems it will be hard to merge?

**Comment by jonmv** [comment]
I'll merge if the changes are approved?

**Comment by kkraune** [comment]
👍

**Pull request merged**